site/content/docs/main/file-system-backup.md

```diff
@@ -473,6 +491,46 @@ kubectl -n velero logs DAEMON_POD_NAME
 **NOTE**: You can increase the verbosity of the pod logs by adding `--log-level=debug` as an argument
 to the container command in the deployment/daemonset pod template spec.
 
+### Verifying backup methods used
+
+To understand which backup method was used for your volumes:
+
+1. Check if volumes were skipped for FSB:
+   ```bash
+   velero backup logs BACKUP_NAME | grep "skipped PVs"
+   ```
+   This will show volumes opted out of FSB, which may still be backed up via snapshots.
+
+2. Verify volume snapshots were created:
+   ```bash
+   velero backup describe BACKUP_NAME --details
+   ```
+   Look for the "Velero-Native Snapshots" or "CSI Snapshots" sections.
+
+3. Check PodVolumeBackups for FSB:
+   ```bash
+   kubectl -n velero get podvolumebackups -l velero.io/backup-name=BACKUP_NAME
+   ```
+
+**Note:** A volume appearing in the "skipped PVs" summary doesn't mean it wasn't backed up - it may have been backed up via volume snapshot instead.
+
+## Backup Method Decision Flow
+
+When Velero encounters a volume during backup, it follows this decision flow:
+
+1. **Is the volume opted out of FSB?** (via `backup.velero.io/backup-volumes-excludes`)
+   - Yes → Skip FSB, attempt volume snapshot (if configured)
+   - No → Continue to step 2
+
+2. **Is the volume opted in for FSB?** (via `backup.velero.io/backup-volumes` or `--default-volumes-to-fs-backup`)
+   - Yes → Perform FSB, skip volume snapshot
+   - No → Attempt volume snapshot (if configured)
+
+3. **For volume snapshots to succeed:**
+   - VolumeSnapshotLocation must be configured
```

Comment:
CSI snapshots must be enabled for CSI volume or compatible VolumeSnapshotLocation must be configured.